mandatory/2-function-creation.js

```diff
@@ -25,7 +48,22 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+
+function tidyUpStrings(arrayOfStrings) {
+
+  var freshArray = [];
+
+  for ( i = 0 ; i < arrayOfStrings.length ; i++) {
+    var toEdit = arrayOfStrings[i];
+    var toTrim = toEdit.trim();
+    var toReplace = toTrim.replace('/','');
+    var toLower = toReplace.toLowerCase();
+    freshArray.push(toLower);
+  }
+  return freshArray;
+}
+
+
```

Comment:
very nice - you may have some more ideas how you could do this following the most recent class (hint: `map` could help)